s3torchconnector/tst/unit/test_s3_client.py

```diff
@@ -17,6 +18,10 @@
 TEST_KEY = "test-key"
 TEST_REGION = "us-east-1"
 S3_URI = f"s3://{TEST_BUCKET}/{TEST_KEY}"
+PYTHON_VERSION = (
+    f"{sys.version_info.major}.{sys.version_info.minor}.{sys.version_info.micro}"
+)
+DEFAULT_USER_AGENT = f"s3torchconnector/{__version__} Python/{PYTHON_VERSION}"
```

Comment:
To be in line with other AWS clients (like the AWS CLI), I'd recommend the format `lang/python#x.y.z`.